automations/js/src/last_week_tonight.mjs

```diff
@@ -18,7 +18,7 @@ import { escapeHtml } from './html.mjs'
 const pat = process.env.ACCESS_TOKEN
 /** the username for the Make site account making the post */
 const username = process.env.MAKE_USERNAME
-/** the application password for the Make site */
+/** the application password, not login password, for the Make site */
 const password = process.env.MAKE_PASSWORD
```

Comment:
Why do we still need it for the weekly updates :thinking:

Reply:
The weekly updates workflow needs to post to the Make site, which is an authenticated operation.

This job only fetches the translations, and does not need to be authenticated (it used to be, but now it's opened up).

Reply:
Ah! Gotcha, I didn't realise this wasn't related to the translations :+1: